src/utils/QCCalculator.cpp

```diff
@@ -396,29 +391,48 @@ class TOPPQCCalculator :
     at.colTypes.push_back("MS:1000285");
     UInt max = 0;
     Size below_10k = 0;
-    for (Size i = 0; i < exp.size(); ++i)
-    {
-      if (exp[i].getMSLevel() == 1)
+    std::vector<OpenMS::Chromatogram> chroms = exp.getChromatograms();
+    if (chroms.size() == 1) //real TIC from the mzML
```

Comment:
Aha, no that's done implicitly only - if chromatogram is the only one ...
There hasn't yet been an attempt to QC SRMfiles or the like with several chromatograms so the decision was pretty simpleton. I guess I can built that in.